MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -434,4 +380,11 @@ open class RouteStepProgress: NSObject {
      The distance in meters the user is to the next intersection they will pass through.
      */
     public var userDistanceToUpcomingIntersection: CLLocationDistance?
+    
+    public var voiceInstructionIndex = 0
+    
+    public var voiceInstruction: SpokenInstruction? {
```

Comment:
Rename to `currentSpokenInstruction` for consistency with `currentStep`, `currentLeg`, etc.